integration/integration_test.go

```diff
@@ -136,6 +137,28 @@ func TestLogs(t *testing.T) {
 		t.Fatal("timed out waiting for logs")
 	case <-logsDone:
 	}
+
+	// Wait for the container to exit
+	client, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+	require.NoError(t, err)
+	require.Eventually(t, func() bool {
+		status, err := client.ContainerInspect(ctx, ctrID)
+		if !assert.NoError(t, err) {
+			return false
+		}
+		return !status.State.Running
+	}, 10*time.Second, time.Second, "container never exited")
```

Comment:
Could 10 seconds be potentially flaky?

Reply:
Potentially, but I think it's relatively unlikely that it will take that long to execute `env` and for the container to exit.